larreco/RecoAlg/TrackMomentumCalculator.cxx

```diff
@@ -485,15 +485,20 @@ namespace trkf {
     double startpoint = 2;
     if (startpoint < min_resolution)
       startpoint = (max_resolution - min_resolution) / 2.; // prevent wrong values
-    if (max_resolution == 0) startpoint = min_resolution;
+    bool fixresolution = false;
+    double maxres = max_resolution;
+    if (max_resolution == 0 || max_resolution == min_resolution) {
```

Comment:
The fix looks reasonable, @hvsouza.  We suggest changing this line to be:

```suggestion
    if (max_resolution == 0 || max_resolution <= min_resolution) {
```

in case there are rounding issues.